tests/settings/e2e/steps/currency-display-settings.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import { When, Then} from 'cucumber';
```

Comment:
Did you execute prettier in this file?

Reply:
@ManusMcCole add a space after "Then" 
`e.g. import { When, Then } from 'cucumber'; `
or run `yarn prettier:format`